Src/StackifyLib.log4net/StackifyLib.log4net.csproj

```diff
@@ -1,9 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+	<PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.3'">
```

Comment:
you can move these (and also common package versions) out to a `Directory.Build.props` file like [this](https://github.com/aspnet/SignalR/blob/dev/Directory.Build.props) so you keep the definitions in a single place.